### PR TITLE
[home-assistant] Add bitnami influxdb chart as optional dependency

### PR DESCRIPTION
**Pull request by onedr0p**
#### Special notes for your reviewer:

This is using the [bitnami helm chart](https://github.com/bitnami/charts/tree/master/bitnami/influxdb) instead of the [official helm chart](https://github.com/influxdata/helm-charts/tree/master/charts/influxdb). 

The main reason I went with bitnami is because they have support for either deploying a deployment or a highly available STS. Bitnami chart in general seems a bit more customizable.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/home-assistant/values.yaml [hunk 1]
```diff
@@ -339,6 +339,8 @@ esphome:
       mountPath: /config/secrets.yaml
       subPath: secrets.yaml
 
+# Enabled mariadb
```
Comment (bjw-s) [inline charts/home-assistant/values.yaml:342]:
```suggestion
# Enable mariadb
```

File: charts/home-assistant/values.yaml [hunk 2]
```diff
@@ -339,6 +339,8 @@ esphome:
...
+# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/mariadb
```
Comment (bjw-s) [inline charts/home-assistant/values.yaml:343]:
```suggestion
# ... for configuration options see https://github.com/bitnami/charts/tree/master/bitnami/mariadb
```

File: charts/home-assistant/values.yaml [hunk 3]
```diff
@@ -353,6 +355,9 @@ mariadb:
     persistence:
       enabled: false
       # storageClass: ""
+
+# Enabled postgres
```
Comment (bjw-s) [inline charts/home-assistant/values.yaml:359]:
```suggestion
# Enable postgres
```

File: charts/home-assistant/values.yaml [hunk 4]
```diff
@@ -353,6 +355,9 @@ mariadb:
...
+# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/postgresql
```
Comment (bjw-s) [inline charts/home-assistant/values.yaml:360]:
```suggestion
# ... for configuration options see https://github.com/bitnami/charts/tree/master/bitnami/postgresql
```

File: charts/home-assistant/values.yaml [hunk 5]
```diff
@@ -364,6 +369,19 @@ postgresql:
     enabled: false
     # storageClass: ""
 
+# Enable influxdb
+# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/influxdb
```
Comment (bjw-s) [inline charts/home-assistant/values.yaml:373]:
```suggestion
# ... for configuration options see https://github.com/bitnami/charts/tree/master/bitnami/influxdb
```

File: charts/home-assistant/values.yaml [hunk 6]
```diff
@@ -364,6 +369,19 @@ postgresql:
...
+  enabled: false
+  architecture: standalone
+  database: home_assistant
+  authEnabled: false
+  persistence:
+    enabled: false
+    # storageClass: ""
+    # size: 8Gi
+
+
```
Comment (bjw-s) [inline charts/home-assistant/values.yaml:384]: Duplicate newline?

**Review by billimek — APPROVED** [review]

**Review by bjw-s — COMMENTED** [review]
Just a few minor things

**Comment by onedr0p** [comment]
Looks like the changes went in @bjw-s, we can fix these up in a future PR (if ever) haha

**Comment by bjw-s** [comment]
No problem, it was mostly some textual stuff

**Comment by billimek** [comment]
oops, sorry you two! I only now realized that @onedr0p requested a review from @bjw-s, too.  I'll pay more attention next time!

**Pull request merged**